### DISCUSSION TITLE
pp/score submit system revamp

### DISCUSSION THREAD

**Discussion by Benjminland** [ppy/osu#17388] [Ideas] [answered]
we all know osu pp submit system is werid and makes everyone throw keyboards
the system right now can only submit the score when it is the personal highest **score**, therefore if i got a higher pp but my score is a bit lower than personal best, i would be stolen a higher pp and rank up
i wish the new system can submit scores by pp, not the score

**Comment by bdach** [comment, marked as answer]
The phenomenon of losing pp due to setting a new personal best score with less pp can be addressed after ongoing work on the submission infrastructure is complete.

**Reply by Rekunan** [reply]
and if you want to, you can read up abt it here: https://github.com/ppy/osu-infrastructure/blob/master/score-submission.md
